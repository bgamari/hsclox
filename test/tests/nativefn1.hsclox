print abs(-5.4);
print getEnv("nope");
