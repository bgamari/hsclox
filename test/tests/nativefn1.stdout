== <script> ==
0000    1 OP_GET_GLOBAL       0 'abs
0002    | OP_CONSTANT         1 '5.4
0004    | OP_NEGATE
0005    | OP_CALL             1
0007    | OP_PRINT
0008    2 OP_GET_GLOBAL       2 'getEnv
0010    | OP_CONSTANT         3 'nope
0012    | OP_CALL             1
0014    | OP_PRINT
0015    3 OP_NIL
0016    | OP_RETURN
5.4
nil
